### PR TITLE
CSCOIVA-1277 Localization to organization name and region

### PR DESCRIPTION
**Pull request by MakJer**

### CODE REVIEW

File: src/scenes/Esittelijat/UusiAsiaEsidialog.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ import {
   Button
 } from "@material-ui/core";
 import { useOrganisations } from "../../stores/organisations";
-import { map, prop, sortBy } from "ramda";
+import * as R from "ramda";
```
Comment (rapitkan) [inline src/scenes/Esittelijat/UusiAsiaEsidialog.js:14]: Why do you like to import everything from Ramda? I'm not delighted about the change but it doesn't prevent approving this PR.

File: src/scenes/Esittelijat/modules/asiatUtils.js [hunk 2]
```diff
@@ -39,19 +39,20 @@ const generateAsiatTableHeaderStructure = t => {
...
-const getJarjestajaNimiFromHakemus = hakemus => {
-  return R.path(["jarjestaja", "nimi", "fi"], hakemus) || "";
+const getJarjestajaNimiFromHakemus = (hakemus, locale) => {
+  const nimi = R.path(["jarjestaja", "nimi"], hakemus);
+  return R.prop(locale, nimi) || R.last(R.values(nimi));
 };
 
 // Generates common row data for all Asiat-tables
-export const generateAsiaTableRows = (row, i, t) => {
+export const generateAsiaTableRows = (row, i, {formatMessage, locale}) => {
```
Comment (rapitkan) [inline src/scenes/Esittelijat/modules/asiatUtils.js:55]: i can be removed. It's not used.

File: src/scenes/Esittelijat/components/PaatetytAsiat.js [hunk 3]
```diff
@@ -33,11 +33,11 @@ const PaatetytAsiat = () => {
     return !!muutospyynnot.paatetyt && muutospyynnot.paatetyt.fetchedAt
       ? generatePaatetytAsiatTableStructure(
           muutospyynnot.paatetyt.data,
-          intl.formatMessage,
+          intl,
           history
         )
       : [];
-  }, [history, intl.formatMessage, muutospyynnot.paatetyt]);
+  }, [history, intl, muutospyynnot.paatetyt]);
```
Comment (rapitkan) [inline src/scenes/Esittelijat/components/PaatetytAsiat.js:40]: formatMessage is the only method / property that generatePaatetytAsiatTableStructure needs. Why should it take the whole intl object as a parameter?

File: src/scenes/Esittelijat/modules/asiatUtils.js [hunk 4]
```diff
@@ -65,20 +66,21 @@ export const generateAsiaTableRows = (row, i, t) => {
     },
     [
       { text: row.asianumero },
-      { text: t(common["asiaTypes.lupaChange"]) }, // Only one type known in system at this juncture
-      { text: getJarjestajaNimiFromHakemus(row) },
-      { text: getMaakuntaNimiFromHakemus(row) },
+      { text: formatMessage(common["asiaTypes.lupaChange"]) }, // Only one type known in system at this juncture
```
Comment (rapitkan) [inline src/scenes/Esittelijat/modules/asiatUtils.js:69]: Good! Replacing t with formatMessage makes the syntax clearer.

File: src/scenes/Esittelijat/modules/asiatUtils.js (hunk 2 above)
Comment (MakJer) [reply src/scenes/Esittelijat/modules/asiatUtils.js:55]: for a brief moment this comment was quite confusing until i realised _i_ references to a parameter  :)

File: src/scenes/Esittelijat/UusiAsiaEsidialog.js (hunk 1 above)
Comment (MakJer) [reply src/scenes/Esittelijat/UusiAsiaEsidialog.js:14]: i thought code would be more readable when ramda functions are on their own namespace. side effect is that everything is imported and tree shaking and other optimizations cannot be run as effectively as before.

Maybe i will revert these changes

File: src/scenes/Esittelijat/components/PaatetytAsiat.js (hunk 3 above)
Comment (MakJer) [reply src/scenes/Esittelijat/components/PaatetytAsiat.js:40]: locale is needed when showing organization name and region

File: src/scenes/Esittelijat/modules/asiatUtils.js (hunk 2 above)
Comment (rapitkan) [reply src/scenes/Esittelijat/modules/asiatUtils.js]: This made my day. :D

**Pull request merged**